1906-12-01.xml

```diff
@@ -63,15 +63,11 @@
                     <head>Orient-Royal Mail Line.</head>
                     <p>REDUCED SUMMER FARES FROM MAY TO OCTOBER INCLUSIVE.</p>
                     <p>OUTWARD to AUSTRALIA.</p>
-                    <p><name>R.M.S. "Ortona"</name> will leave Suez about <date when="1906-12-14"
-                            >November 30</date></p>
-                    <p><name>R.M.S "Ormuz"</name> will leave Suez about <date when="1906-10-19"
-                            >December 14</date>.</p>
+                    <p><name>R.M.S. "Ortona"</name> will leave Suez about <date when="1906-12-14">November 30</date></p>
```

Comment:
note that the `when=""` attributes need to be updated here. For example, this one should be `when-"1906-11-30"`